client/src/modules/templates/bhStockEntryExitType.tmpl.html

```diff
@@ -1,24 +1,23 @@
-<div ng-if="type.isAllowed" class="col-md-3 col-xs-6" ng-repeat="type in $ctrl.entryExitTypeList | filter:{isEntry : $ctrl.isEntry}">
```

Comment:
Good catch by defining `entryTypes` or `exitTypes` in the controller instead of filtering in the view 👍